### PR TITLE
Category basic view added and some general functionality improved

### PR DESCRIPTION
**Pull request by svelaf71**

### CODE REVIEW

File: MVCapplication/src/main/webapp/WEB-INF/views/categories.jsp [hunk 1]
```diff
@@ -12,4 +13,32 @@
...
+                        <th class="text-center">Name</th>
+                        <th class="text-center">Edit</th>
+                        <th class="text-center">Delete</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <c:forEach items="${categories}" var="category">
+                        <tr class="gradeA">
+                            <td><c:out value="${category.id}"></c:out></td>
+                            <td><c:out value="${category.name}"></c:out></td>
```
Comment (kucherif) [inline MVCapplication/src/main/webapp/WEB-INF/views/categories.jsp]: you can imit c:out. 
${category.name} will be enough

Comment (svelaf71) [inline MVCapplication/src/main/webapp/WEB-INF/views/categories.jsp]: You mean I don't have to pull ${category.id}?

Comment (kucherif) [inline MVCapplication/src/main/webapp/WEB-INF/views/categories.jsp]: I mean that 
` <td><c:out value="${category.name}"></c:out></td>`
and 
` <td>${category.name}"></td>`
doing the same thing in this case

Comment (svelaf71) [inline MVCapplication/src/main/webapp/WEB-INF/views/categories.jsp]: got it, tnx

**Pull request merged**